hbw/app/controllers/hbw/tasks_controller.rb

```diff
@@ -24,11 +24,6 @@ def list
       @tasks = widget.task_list(current_user_identifier, entity_class)
     end
 
-    def count
-      @task_count = widget.task_count(current_user_identifier, entity_class)
-      @task_count_unassigned = widget.task_count_unassigned(current_user_identifier, entity_class)
```

Comment:
Remove same-named methods from `adapter.rb`, `Task` model and delegators from `widget.rb` as well, please.